File: src/LiWiMus.API/Infrastructure/Attributes/AuthorizeWithJwtAttribute.fs
namespace LiWiMus.API.Infrastructure.Attributes

open Microsoft.AspNetCore.Authorization
open OpenIddict.Validation.AspNetCore

type AuthorizeWithJwtAttribute() as this =
    inherit AuthorizeAttribute()
    do this.AuthenticationSchemes <- OpenIddictValidationAspNetCoreDefaults.AuthenticationScheme

File: src/LiWiMus.API/Attributes/AuthorizeWithJwtAttribute.cs
using Microsoft.AspNetCore.Authorization;
using OpenIddict.Validation.AspNetCore;

namespace LiWiMus.API.Attributes;

public class AuthorizeWithJwtAttribute : AuthorizeAttribute
{
    public AuthorizeWithJwtAttribute()
    {
        AuthenticationSchemes = OpenIddictValidationAspNetCoreDefaults.AuthenticationScheme;
    }
}